Dockerfile

```diff
@@ -1,11 +1,10 @@
-FROM golang:1.11-alpine3.9 as builder
-RUN mkdir -p /go/src/github.com/mendersoftware/useradm
+FROM golang:1.14-alpine3.12 as builder
```

Comment:
The reason why the unit/acceptance tests are failing now is because of a semantic error in `main_test.go` (not allowed in go>=1.13).
Move the call to `flag.Parse()` from `init()` -> `TestRunMain()`.

Reply:
yaaay thank you 🥳